ggml/src/ggml-blas.cpp

```diff
@@ -8,11 +8,12 @@
 #   include <Accelerate/Accelerate.h>
 #elif defined(GGML_BLAS_USE_MKL)
 #   include <mkl.h>
+#elif defined(BLIS_ENABLE_CBLAS)
+#   include <blis.h>
+#elif defined(NVPL_ENABLE_CBLAS)
+#   include <nvpl_blas.h>
```

Comment:
I am not sure if I understand the purpose of this change. `BLIS_ENABLE_BLAS` is defined in `cblas.h` when it is provided by blis, so it used later to determine availability of the `bli_thread_set_num_threads` function. As it is, `blis.h` will never get included because this is not defined at this point. Admittedly, this is not very reliably because if multiple BLAS libraries are installed, it is possible that the one providing `cblas.h` is not the library linking to, so it might be better to add defines in the `Makefile` similar to `GGML_BLAS_USE_MKL` for other libraries, such as `GGML_BLAS_USE_BLIS` and `GGML_BLAS_USE_NVPL`, and use that to include the library header. In short, what I suggest is:
- Define `GGML_BLAS_USE_BLIS` when using `GGML_BLIS` in the `Makefile` and use it to determine if `blis.h` needs to be included
- Rename `NVPL_ENABLE_CBLAS` to `GGML_BLAS_USE_NVPL` in the `Makefile`